src/tasks/daily.ts

```diff
@@ -164,6 +166,53 @@ function voterSetup(): void {
   visitUrl(`choice.php?option=1&whichchoice=1331&g=${monsterVote}&local[]=${init}&local[]=${init}`);
 }
 
+function chibiSetup(): void {
+  if (!have($item`ChibiBuddy™ (on)`) && !have($item`ChibiBuddy™ (off)`)) return;
+  // We need to name our buddy when we turn it on!
+  const chibiNames = [
+    "SSBBHax",
+    "Frasier Crane",
+    "William",
+    "Gamuel Sauce",
+    "Axis Shadowbaenimus",
```

Comment:
Interesting pick

Reply:
I'll admit my picks were almost completely random. Very open to suggestions ahaha